src/api/database/shard.rst

```diff
@@ -0,0 +1,220 @@
+    :code 500: Internal server error or timeout
+
+    **Request**:
+
+    .. code-block:: http
+
+        GET /db/_shards HTTP/1.1
+        Accept: */*
+        Host: localhost:5984
+
+    **Response**:
+
+    .. code-block:: http
+
+        HTTP/1.1 200 OK
+        Cache-Control: must-revalidate
+        Content-Length: 621
+        Content-Type: application/json
+        Date: Fri, 18 Jan 2019 19:55:14 GMT
+        Server: CouchDB/2.4.0 (Erlang OTP/19)
+
+        {
+          "shards": {
+            "00000000-1fffffff": [
+              "couchdb@node1.example.com",
+              "couchdb@node2.example.com",
+              "couchdb@node3.example.com"
+            ],
+            "20000000-3fffffff": [
+              "couchdb@node1.example.com",
+              "couchdb@node2.example.com",
+              "couchdb@node3.example.com"
+            ],
+            "40000000-5fffffff": [
+              "couchdb@node1.example.com",
+              "couchdb@node2.example.com",
+              "couchdb@node3.example.com"
+            ],
+            "60000000-7fffffff": [
+              "couchdb@node1.example.com",
+              "couchdb@node2.example.com",
+              "couchdb@node3.example.com"
+            ],
+            "80000000-9fffffff": [
+              "couchdb@node1.example.com",
+              "couchdb@node2.example.com",
+              "couchdb@node3.example.com"
+            ],
+            "a0000000-bfffffff": [
+              "couchdb@node1.example.com",
+              "couchdb@node2.example.com",
+              "couchdb@node3.example.com"
+            ],
+            "c0000000-dfffffff": [
+              "couchdb@node1.example.com",
+              "couchdb@node2.example.com",
+              "couchdb@node3.example.com"
+            ],
+            "e0000000-ffffffff": [
+              "couchdb@node1.example.com",
+              "couchdb@node2.example.com",
+              "couchdb@node3.example.com"
+            ]
+          }
+        }
+
+.. _api/db/shards/doc:
+
+==============================
+``/db/_shards/doc``
+==============================
+
+.. http:get:: /{db}/_shards/{docid}
+    :synopsis: Returns the specific shard in which a document is stored
+
+    Returns information about the specific shard into which a given document
+    has been stored, along with information about the nodes on which that
+    shard has a replica.
+
+    :param db: Database name
+    :param docid: Document ID
+    :<header Accept: - :mimetype:`application/json`
+                     - :mimetype:`text/plain`
+    :>header Content-Type: - :mimetype:`application/json`
+                           - :mimetype:`text/plain; charset=utf-8`
+    :>json string range: The shard range in which the document is stored
+    :>json array nodes: List of nodes serving a replica of the shard
+    :code 200: Request completed successfully
+    :code 401: Read privilege required
+    :code 404: Database or document not found
+
```

Comment:
In general, I question the value of mentioning '500' errors in API documentation, since in theory, ti could happen anywhere, but since we do seem to mention that on other endpoints (including the one above), is it worth putting here?